### ISSUE TITLE
duplicate records

### ISSUE THREAD

**Issue by ZymanteM** [atviriduomenys/manifest#1575]

**Comment by sirex** [comment]
@Marius-prog, @DataAIchemist who can fix that?

**Comment by DataAIchemist** [comment]
By writing this query https://get.data.gov.lt/datasets/gov/lsd/cl/ja_asmenys/JuridinisAsmuo?ja_kodas=%22222359310%22,
I see no duplicate values. However, it looks like it's .CSV file writing problem.  @sirex  who can fix it?

**Comment by sirex** [comment]
@DataAIchemist everything is ok with csv export:

```python
>>> df = pd.read_csv("https://get.data.gov.lt/datasets/gov/lsd/cl/ja_asmenys/JuridinisAsmuo/:format/csv")
>>> df[df['ja_kodas'] == 222359310].shape
(1, 12)
```

CSV file contains exactly one row as it is shown in HTML export.

But there is one concerning thing, that there are several different `_id`'s for the same `ja_kodas`. `_id` should never change. Also in the original post, we see, that two `_id`s are different, that means, there are duplicates. And currently we have a third `_id`.

Also we can see, that object [7d080684-3af5-4b16-a015-ef32499c867e](https://get.data.gov.lt/datasets/gov/lsd/cl/ja_asmenys/JuridinisAsmuo/7d080684-3af5-4b16-a015-ef32499c867e/:changes) was created and then was deleted.

So basically, I see, that there was an issue with duplicates, caused by incorrect publishing. CSV export works fine.

But now I see, that there are no duplicates and it seems, that issue is fixed.

```python
>>> (df['ja_kodas'].value_counts() > 1).sum()
0
```

Some I'm closing this issue. @ZymanteM thanks for the report.
